### PR TITLE
Use more ES6 syntax (thx ember-watson)

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: blueprints/locale/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   description: 'Generates new i18n locale directory with config and translations files.',
 
-  beforeInstall: function(options) {
```
Comment (jamesarosen) [inline blueprints/locale/index.js]: This requires Node 6, doesn't it?

File: index.js [hunk 2]
```diff
@@ -7,15 +7,15 @@ module.exports = {
   name: 'ember-i18n',
   isLocalizationFramework: true,
 
-  init: function() {
+  init() {
```
Comment (jamesarosen) [inline index.js]: This would also require a fairly recent version of node

Comment (rwjblue) [inline index.js]: It is Node 4+ (and Node 0.12 is EOL)...

File: blueprints/locale/index.js (hunk 1 above)
Comment (rwjblue) [inline blueprints/locale/index.js]: Node 4 or higher

**Comment by jamesarosen** [comment]
+1

**Pull request merged**